crates/precompile/Cargo.toml

```diff
@@ -46,6 +46,11 @@ bn = { package = "substrate-bn", version = "0.6", default-features = false }
 # KZG point evaluation precompile
 c-kzg = { version = "1.0.2", default-features = false, optional = true }
 
+# Optionally use `kzg-rs` for a pure Rust implementation of KZG point evaluation.
+kzg-rs = { git = "https://github.com/0xWOLAND/kzg-rs", default-features = false, features = [
```

Comment:
needs to be published

Reply:
Thanks! I switched to the published version